Trimble.Modus.Components/Controls/TabBar/TMTabbedPage.cs

```diff
@@ -0,0 +1,347 @@
+﻿using System.Collections;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using Trimble.Modus.Components.Constant;
+using Trimble.Modus.Components.Helpers;
+
+namespace Trimble.Modus.Components;
+
+[ContentProperty(nameof(TabItems))]
+public partial class TMTabbedPage : ContentPage
+{
+    #region Private Fields
+    private Grid mainContainer;
+    private Grid tabStripContainer;
+    private CarouselView contentContainer;
+    private List<double> contentWidthCollection;
```

Comment:
What is the use of `contentWidthCollection`? Doesn't see to be used anywhere

Reply:
Removed